### ISSUE TITLE
Not able to connect RAW instance

### ISSUE THREAD

**Issue by netzwerkgoettin** [Checkmk/grafana-checkmk-datasource#55]
Hi,

today I checked out this plugin (e59e5b043927c4cb5cc1b2fca803e3cf0e83e42a). I integrated it into an existing Grafana (8.4.4) which is running as a Docker container.

I try to connect it to a Checkmk RAW test instance running also within Docker (Checkmk Raw Edition 2.1.0-2022.03.31) and this is what I get:

```
Checkmk exception: Currently not supported with this Checkmk Edition
Traceback (most recent call last): File "/omd/sites/cmk/lib/python3/cmk/gui/plugins/webapi/grafana.py",
line 118, in _get_combined_graph_identifications from cmk.gui.cee.plugins.metrics.graphs
import ( ModuleNotFoundError: No module named 'cmk.gui.cee' During handling of
the above exception, another exception occurred: Traceback (most recent call
last):
File "/omd/sites/cmk/lib/python3/cmk/gui/webapi.py", line 116, in page_api resp = _execute_action(api_call, request_object)
File "/omd/sites/cmk/lib/python3/cmk/gui/webapi.py", line 207, in _execute_action return _execute_action_no_lock(api_call, request_object)
File "/omd/sites/cmk/lib/python3/cmk/gui/webapi.py", line 221, in _execute_action_no_lock "result": api_call["handler"](request_object),
File "/omd/sites/cmk/lib/python3/cmk/gui/plugins/webapi/grafana.py", line 123, in _get_combined_graph_identifications raise
MKGeneralException(_("Currently not supported with this Checkmk Edition")) cmk.utils.exceptions.MKGeneralException:
Currently not supported with this Checkmk Edition
```

According to [the docs](https://docs.checkmk.com/latest/en/grafana.html): *„Beginning with version 2.0.0, it is possible in all Checkmk editions to directly address Checkmk as a data source in Grafana*“ which in my understanding includes RAW – at least it wouldn't make sense to offer a dropdown containing „RAW“ and „Enterprise“ when RAW won't work.

So please help me: what am I missing?

Regards,
  Marianne

**Comment by BenediktSeidl** [comment]
This looks like a bug: When using the "Save & test" button on the "Data Source" page, an enterprise endpoint is used to check if it's possible to fetch data from checkmk, even though "RAW Edition" is selected. I've create an internal ticket (`CMK-10237`) to track and fix this bug.
The good news: you can ignore this error message, the data source is working although the save test failed.

**Comment by netzwerkgoettin** [comment]
Hi,

thank you for your reply!
I can confirm it's possible to choose this data source in a panel, but I'm not able to work with it: while „Site“ let me choose between `All sites` and `Local site cmk`, „Hostname“ and „Service“ do not have (access to any) entries.

Perhaps useful for your debugging, too :)

Regards,
  Marianne

<img width="991" alt="Screenshot 2022-04-06 at 10 15 38" src="https://user-images.githubusercontent.com/11242139/161929242-04f2db1a-1775-41fa-a990-105ab15b7675.png">

**Comment by BenediktSeidl** [comment]
The "Save & test" error message on raw editions should now be fixed. If you want to test this yourself you will have to do a `yarn install` and `yarn build`. I also saw some drop down problems and created #56. I will come back to your drop down issue when i resolved #56.

**Comment by BenediktSeidl** [comment]
We released version [2.0b3](https://github.com/tribe29/grafana-checkmk-datasource/releases/tag/2.0b3). Can you please try to reproduce your second issue with this release?
Please check if the user used for the grafana connection is able to see at least some hosts? (This should be the case for the automation user in a fresh installation)
Are there any errors in the browser console or grafana log?
